### DISCUSSION TITLE
Hot reload

### DISCUSSION THREAD

**Discussion by jimodelid** [prisma/prisma#4764] [Q&A] [answered]
I can't seem to find any thread about it but it would be awesome to have data update live without reloading the page similar to Firebase and other services, listening to streams, as of now I'm using SQLITE and NEXTJS, it would be cool to see posts update or added in realtime without having to reload the page.

**Comment by ryands17** [comment, marked as answer]
Hey @jimodelid 👋 
Currently Prisma doesn't have any mechanism for real-time data fetching, so it would be great if you could add a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with your use case so that we can look into this :)

**Reply by jimodelid** [reply]
Thank you, I’ve added a request on the link you posted. :D

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
